File: .cpp/Program/Keyboard.cpp
#pragma once
#include <bits/stdc++.h>
#include "Device.cpp"
using namespace std;

class Keyboard : public Device {
    private:
        string connectionType;
        string switchType;

    public:
        Keyboard () { }
         
        Keyboard (string brand, string model, string connectionType, string switchType) : Device (brand, model) {
            this->connectionType = connectionType;
            this->switchType = switchType;
        }

        void setConnectionType (string connectionType) { this->connectionType = connectionType; }
        void setSwitchType (string switchType) { this->switchType = switchType; }

        string getConnectionType () { return connectionType; }
        string getSwitchType () { return switchType; }

        void info () override {
            cout << "Keyboard" << endl;
            cout << "| Brand             : " << getBrand() << endl;
            cout << "| Model             : " << getModel() << endl;
            cout << "| Connection Type   : " << connectionType << endl;
            cout << "| Switch Type       : " << switchType << endl;
        }

        ~Keyboard () { }
};

File: .py/Program/Keyboard.py
from Device import Device

class Keyboard (Device):
    __connectionType = ""
    __switchType = ""

    def __init__ (self, brand = "", model = "", connectionType = "", switchType = ""):
        super ().__init__ (brand, model)
        self.__connectionType = connectionType
        self.__switchType = switchType

    def setConnectionType (self, connectionType):
        self.__connectionType = connectionType

    def setSwitchType (self, switchType):
        self.__switchType = switchType

    def getConnectionType (self):
        return self.__connectionType
    
    def getSwitchType (self):
        return self.__switchType
    
    def info(self):
        print("| Keyboard")
        print(f"| Brand             : {self.getBrand()}")
        print(f"| Model             : {self.getModel()}")
        print(f"| Connection Type   : {self.getConnectionType()}")
        print(f"| Switch Type       : {self.getSwitchType()}")